### ISSUE TITLE
Elements must only use allowed ARIA attributes

### ISSUE THREAD

**Issue by seanforyou23** [patternfly/patternfly-next#617]
Several component demos exhibit this same issue. Details related to this accessibility violation can be found here: https://dequeuniversity.com/rules/axe/3.0/aria-allowed-attr

Below are the URLs that illustrate the problem:
- /components/OptionsMenu/examples-full/
- /components/Pagination/examples-full/
- /components/Toolbar/examples-full/

This is a `critical` a11y issue.

Related to https://github.com/patternfly/patternfly-next/issues/541

To learn more about how to build and test locally, check out: https://github.com/patternfly/patternfly-next#testing-for-accessibility 

**As part of this issue, please complete the following items.** These will help reviewers know what issues were addressed in the PR, and also will help us keep our total a11y violation count down.

1. Run the a11y audit, copy the issues from the audit that were fixed in the PR, and paste them into the PR description. 
    - Instructions for running the audit are here: https://github.com/patternfly/patternfly-next#testing-for-accessibility
    - Issues that are returned in the terminal look like this. Find the issues you are addressing and copy the following information provided for them or take a screencap to include in the PR.
      ```
      1 error found in: /upgrade-examples/AlertUpgradeExamples/examples-full/

      1) Buttons must have discernible text -- https://dequeuniversity.com/rules/axe/3.1/button-name -- impact: critical
      
      -------html violation instances-------

      <button class="btn btn-link dropdown-toggle" type="button" id="dropdownKebabRight" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
        <span class="fa fa-ellipsis-v"></span>
      </button> 
      ```
2. Reduce the threshold by number of issues being addressed 
    - This update is made [here](https://github.com/patternfly/patternfly-next/blob/master/test-a11y/config.js#L2)
    - The new value should be derived by: 
      - subtracting the number of issues you are addressing in the PR (e.g. the line that looks like this in the info you copied from the audit report in item 1 above):
        `5 errors found in: /components/Toolbar/examples-full/`
      - from current violations (e.g. this line in the audit report, near the end):
        `20 violations in total`

**Comment by rachael-phillips** [comment]
I'm following up on this issue. Any updates here?

**Comment by seanforyou23** [comment]
@rachael-phillips looks like the original issue in the Form component was resolved, but it's crept back up in three other components since then. I updated the issue accordingly.

**Comment by rachael-phillips** [comment]
Thanks!
